modules/rosa-revoke-objects.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * rosa_cli/rosa-manage-objects-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-revoke-objects_{context}"]
+= Revoke objects
+
+This section describes the `revoke` commands for clusters and resources.
+
+[id="rosa-revoke-break-glass-credential_{context}"]
+== revoke-break-glass-credential
+
+Revokes all break glass credentials from a specified hosted control plane cluster with external authentication enabled.
+
+.Syntax
+[source,terminal]
+----
+$ rosa revoke break-glass-credential --cluster=<cluster_name> | <cluster_id>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|Required: The name or ID (string) of the cluster from which the break glass credentials will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+
+|--yes
+|Automatically answers `yes` to confirm the operation.
+|===
+
+.Example
+Revoke a break glass credential from a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa revoke break-glass-credential -c  --cluster=mycluster
```

Comment:
Here the command must be "rosa revoke break-glass-credential --cluster=mycluster"